deploy-manage/deploy/cloud-enterprise/ece-install-offline-images.md

```diff
@@ -40,11 +36,17 @@ Enterprise Search is not available in versions 9.0+.
 
 | Required downloads | Minimum required ECE version |
 | --- | --- |
-| [Elasticsearch, Kibana, APM, and Enterprise Search stack pack: 8.17.1](https://download.elastic.co/cloud-enterprise/versions/8.17.1.zip) | ECE 3.0.0<br>(+ Docker 20.10.10+ required for 8.16+) |
-| docker.elastic.co/cloud-release/elasticsearch-cloud-ess:8.17.1 | ECE 3.0.0<br>(+ Docker 20.10.10+ required for 8.16+) |
-| docker.elastic.co/cloud-release/kibana-cloud:8.17.1 | ECE 3.0.0 |
-| docker.elastic.co/cloud-release/elastic-agent-cloud:8.17.1 | ECE 3.0.0 |
-| docker.elastic.co/cloud-release/enterprise-search-cloud:8.17.1 | ECE 3.0.0 |
+| [Elasticsearch, Kibana, APM, and Enterprise Search stack pack: 9.0.0](https://download.elastic.co/cloud-enterprise/versions/9.0.0.zip) | ECE 4.0.0<br>(+ Docker 20.10.10+ required for 8.16+) |
```

Comment:
added this row, but I'm not sure if the note in parentheses is accurate or maybe it should be `Docker 25 is required for ECE 4.0`

Reply:
Since the column is explicitly about the "Minimum required ECE version", I'm not sure if the content between parentheses is much needed, or even confusing? People should refer to the compatibility matrix of their ECE version whenever they install or upgrade ECE.


<img width="727" alt="image" src="https://github.com/user-attachments/assets/be9b929c-1a18-46e6-8ff3-f756042eb312" />

Reply:
I think we can remove the parentheses. This additional comment is required because older ECE versions can run lower Docker versions, but then they won't support this stack.

For ECE 4.0 we only allow docker 25 in the first place, so this additional comment is redundant.

Reply:
ok I removed the parentheses comment in d065f975